packages/backend/src/accounting/tigerbeetle/utils.ts

```diff
@@ -60,6 +60,9 @@ export function tbTransferToLedgerTransfer(
     transferRef: fromTigerBeetleId(tbTransfer.user_data_128),
     type: transferTypeFromCode(tbTransfer.code),
     state: state,
-    ledger: tbTransfer.ledger
+    ledger: tbTransfer.ledger,
+    expiresAt: new Date(
+      Number(tbTransfer.timestamp / 1_000_000n) + tbTransfer.timeout * 1000
+    )
```

Comment:
Thinking about this, `expiresAt` should only be set/returned if a `timeout` exists (otherwise, its a [posted](https://docs.tigerbeetle.com/reference/transfer#timeout) transfer by default).